language/move-prover/bytecode/src/stackless_bytecode.rs

```diff
@@ -484,6 +484,7 @@ impl<'env> BytecodeDisplay<'env> {
         Ok(())
     }
 
+    #[allow(clippy::rc_buffer)]
```

Comment:
Can we add this to the x.toml instead of here, that way we know we can go and and clean this up later (if needed)

Reply:
Done, wanted to make it a warn but didn't function as a warn, only error....